test/unit/registry-domain-plugins-detective.js

```diff
@@ -1,6 +1,7 @@
 'use strict';
 
 const expect = require('chai').expect;
+const code = require('../fixtures/mocked-components/package-server-with-plugin');
```

Comment:
@matteofigus what do you think about this? I only added the `data` part and not the full object; you know that I'm a minimalist 😅